build.go

```diff
@@ -429,8 +468,6 @@ func translate() {
 func transifex() {
 	os.Chdir("gui/default/assets/lang")
 	runPrint("go", "run", "../../../../script/transifexdl.go")
```

Comment:
This relative traversing the tree looks strange to me. What about a convention to run commands from root directory and use something like the environment variable PWD (current working directory).

Relative paths looks very fragile to me (get broken by refactorings etc.). Or is that recommended GO convention?